README.md

```diff
@@ -114,6 +110,9 @@ images tagged `janus_aggregator`, `janus_garbage_collector`,
 Pre-built container images are available at
 [us-west2-docker.pkg.dev/divviup-artifacts-public/janus](https://us-west2-docker.pkg.dev/divviup-artifacts-public/janus).
 
+Note that `podman` is not an acceptable substitute for `docker`. There are
+subtle incompatibilities between the two that will cause tests to fail.
```

Comment:
I think this paragraph should go under "Running tests", after `cargo xtask test-docker`.